ceph-client.rb

```diff
@@ -21,6 +21,7 @@ class CephClient < Formula
   depends_on "python3"
   depends_on "sphinx-doc" => :build
   depends_on "yasm"
+  depends_on "lua"
```

Comment:
the same. lua is a dependency of the server side. lemme check.

Reply:
please drop this change, adding "cephfs" to `targets` would suffice.

Reply:
done